.buildkite/scripts/steps/on_merge_ts_refs_api_docs.sh

```diff
@@ -2,6 +2,7 @@
 
 set -euo pipefail
 
+export BAZEL_CACHE_MODE=read-write
```

Comment:
Is this here to provide the cache for Linux?

Reply:
Yep, added a comment